README.md

```diff
@@ -10,7 +10,7 @@ This module retrieves some basic [ACS](https://github.com/byu-oit/aws-acs) infor
 
 ```hcl
 module "acs" {
-  source = "github.com/byu-oit/terraform-aws-acs-info.git?ref=v2.1.0"
+  source = "github.com/byu-oit/terraform-aws-acs-info.git?ref=v2.1.1"
```

Comment:
so if I understand correctly, to make a new release you update the `version` file, but you will still need to update this readme and any examples with the new version as well?

Reply:
That's correct. This just automated the creation of the GitHub release, not the changing of the version number.